frontend/spec/spec_helper.rb

```diff
@@ -45,6 +45,16 @@
 
 require "selenium/webdriver"
 require 'webdrivers'
+Capybara.register_driver :selenium_chrome_headless_docker_friendly do |app|
```

Comment:
Do we need to replicate the config here as well?

Reply:
I'm afraid we need it. They belong to two different sub-systems: backend & frontend. In case I remove it from here it will make tests fail when executed from within the `frontend` directory. I haven't tried it, but it's possible that they pass if executed globaly by `bin/rspec` script. However, it would rely on a very brittle source code load order.

It doesn't happen with `:selenium_chrome_headless` because it's a driver already provided out of the box by selenium.